docs/sources/github/README.md

```diff
@@ -61,4 +61,4 @@ Copy the value of `installationId` and assign it to the `github_installation_id`
 8. Once the certificate has been uploaded, please remove {YOUR DOWNLOADED CERTIFICATE FILE} and `gh_pk_pkcs8.pem` from your computer or store it in a safe place.
 
 ## Reference
-These instructions have been derived from [worklytics-connector-specs](../../../infra/modules/worklytics-connector-specs/main.tf); refer to that for definitive information.
\ No newline at end of file
+These instructions have been derived from [worklytics-connector-specs](../../../infra/modules/worklytics-connector-specs/main.tf); refer to that for definitive information.
```

Comment:
I see infra path changes everywhere to ".../../../../infra", but here is "../../../infra". Just like to be sure it is correct.